.travis.yml

```diff
@@ -5,9 +5,9 @@ script: bash -ex .travis-ci.sh
 env:
   global:
   - UPDATE_GCC_BINUTILS=1
-  - EXTRA_REMOTES="https://github.com/yomimono/mirage-dev.git#remove-console"
+  - EXTRA_REMOTES="https://github.com/mirage/mirage-dev.git"
```

Comment:
since we use the mirage-dev remote, we could just add the PINS below to that remote... instead of using both mechanisms....